CMakeLists.txt

```diff
@@ -39,11 +32,16 @@ set(SOURCES
 add_executable(yapgt ${SOURCES})
 
 # add the link from the opencv libs
-target_link_libraries(yapgt PRIVATE ${OpenCV_LIBS})
+target_link_libraries(yapgt ${OpenCV_LIBS})
 
 # Enable C++ 17 Standard
 set_target_properties(yapgt PROPERTIES
         CXX_STANDARD 17
         CXX_STANDARD_REQUIRED ON
         CXX_EXTENSIONS OFF
-)
\ No newline at end of file
+)
+
+# copy every linked opencv library to the binary directory
+foreach(CVLib ${OpenCV_LIBS})
+    file(COPY ${_OpenCV_LIB_PATH}/${CVLib}${OpenCV_VERSION_MAJOR}${OpenCV_VERSION_MINOR}${OpenCV_VERSION_PATCH}d.dll DESTINATION ${CMAKE_BINARY_DIR})
```

Comment:
Here you copy the d.dll files for the opencv modules. Would this work with a release build? If not, we should keep this in mind, if we want to submit a release build. Otherwise everything looks fine.

Reply:
Good question. Not sure how to try this

Reply:
I think we can ignore this until it becomes necessary.

Reply:
then merge